module.go

```diff
@@ -8,7 +8,7 @@ import (
 	"go.viam.com/rdk/module"
 	"go.viam.com/utils"
 
-	"viam-labs/viam-appliedmotion/st"
+	"viam/viam-appliedmotion/st"
```

Comment:
Can we make this reference to the GitHub Go package address?

Reply:
not sure what that means, use Viam-modules instead of viam? I thought viam was the new namespace when the ownership of the module is transferred.

Reply:
no - the current go package is a local go package, if you notice everything else points to a go package with an address on the web, this one only points to a local package.

This is a nit, if you're uncertain on how to do this next time. My main concern in this pr is with verifying that these build and show the resources correctly.

Reply:
think I'm going to leave as is for now.